packages/esm-tb-app/src/views/tpt/program-management/tabs/tpt-treatment.component.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useConfig } from '@openmrs/esm-framework';
+import { EncounterList, EncounterListColumn, getObsFromEncounter } from '@ohri/openmrs-esm-ohri-commons-lib';
+import { moduleName } from '../../../..';
+
+interface TptTreatmentListProps {
+  patientUuid: string;
+}
+
+const TptTreatmentList: React.FC<TptTreatmentListProps> = ({ patientUuid }) => {
+  const { t } = useTranslation();
+  const { obsConcepts, encounterTypes, formNames } = useConfig();
+  const headerTitle = t('tptTreatment', 'TPT Treatment');
+  const columns: EncounterListColumn[] = useMemo(
+    () => [
+      {
+        key: 'caseId',
+        header: t('caseId', 'Case Id'),
+        getValue: (encounter: any) => {
+          return getObsFromEncounter(encounter, obsConcepts.caseID);
+        },
```

Comment:
I was of the understanding that the treatment form doesn't have this value. Has this changed?

Reply:
it does have that value,i confirmed with Veronica